README.md

```diff
@@ -192,6 +192,6 @@ earthaccess is licensed under the MIT license. See [LICENSE](LICENSE.txt).
 
 <div><img src="https://raw.githubusercontent.com/nsidc/earthdata/main/docs/nsidc-logo.png" width="84px" align="left" text-align="middle"/>
 <br>
- This repository is not actively supported by NSIDC but we welcome issue submissions and pull requests in order to foster community contribution.
+ This repository is supported by a joint effort of NSIDC DAAC, other NASA DAACs, and the Earth science community, and we welcome any contribution in the form of issue submissions, pull requests, or discussions.
```

Comment:
```suggestion
 This repository is supported by a joint effort of NSIDC, NASA DAACs, and the Earth science community, and we welcome any contribution in the form of issue submissions, pull requests, or discussions.
```

After learning more about where my funding has been coming from, I think it might be better to be a little less specific here.

Reply:
Just came to me now, but maybe this sentence would be a great place to also link to our project board or backlog (or a filtered version of the backlog only showing "good first issue" labels) so potential contributors not only know contributions are welcome, but which contributions will be most likely to be accepted.

[This comment](https://github.com/nsidc/earthaccess/issues/428#issuecomment-1905003257) has some thoughts in the last paragraph about why that's important.

Reply:
Great idea. I updated with an additional brief sentence - lmk what you think.

Reply:
Love it, thanks! Marking resolved and following up on @jhkennedy's comment below :)